src/libraries/System.Text.RegularExpressions/tests/RegexMatchTimeoutExceptionTests.cs

```diff
@@ -44,6 +44,7 @@ public void Ctor()
         }
 
         [Fact]
+        [PlatformSpecific(~TestPlatforms.Browser)]
```

Comment:
We use `[ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsBinaryFormatterSupported))]` elsewhere:

https://github.com/dotnet/runtime/blob/423e1646eddcab3ccaf0198b04bd00ddbcafa413/src/libraries/System.Collections/tests/Generic/Comparers/EqualityComparer.Generic.Serialization.Tests.cs#L14-L17

Reply:
Thanks :D